src/components/inquiry_artwork_brick/details.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import TextLink from "../text_link"
+
+export interface DetailsProps extends React.HTMLProps<ArtworkDetails> {
+  artwork: any
```

Comment:
You should extend from `RelayProps` as well and drop this `any` typing to get the actual types listed below https://github.com/artsy/reaction-force/pull/61/files#diff-bb6edb8ba51d5c558582b32f2884d355R88. E.g.

```ts
interface DetailsProps extends RelayProps, React.HTMLProps<ArtworkDetails> {}
```

(PS: I left out the `export` because I don’t see this interface being used outside of this file.)